docs/team/johndoe.md

```diff
@@ -3,7 +3,7 @@ layout: page
 title: John Doe's Project Portfolio Page
 ---
 
-## Project: AddressBook Level 3
+## Project: StoreMando Level 3
```

Comment:
I think we can just delete this file also right? If not,  can just remove the Level 3

Reply:
Yup I will remove this file now. :)